bcrhp/src/bcrhp/pages/steps/Step4_SiteNames.vue

```diff
@@ -100,86 +64,88 @@ const saveOtherName = function () {
 
 const addOtherNameDisabled = ref(false);
 
-let validateFields = false;
-
 const deleteOtherNameCallback = function (index: number) {
     heritageSiteRef.value.otherNames.splice(index, 1);
     updateAddOtherState();
 };
 
-// This needs to be removed - added because ESLint was complaining. Need to figure out
-// configuration so API methods are not
-defineExpose({ isValid });
```

Comment:
I'm wondering if we still need to expose this because we want to validate the step from the top level before proceeding to the next step. It's great that we're validating on a field-by-field bases when entering data but we also need logic to do a full-step validation too.